shared/data/content/react-challenges.ts

```diff
@@ -1,6 +1,16 @@
 import type { IChallenge } from '../types/challenge';
 
 export const reactChallenges = new Map<string, IChallenge>([
+  [
+    'transfer-list',
+    {
+      title: 'Transfer List',
+      link: 'transfer-list',
+      difficulty: 'medium',
+      developer: 'DeePaK-HeeRaKaRi',
```

Comment:
add your details to contributors list as well

Reply:
Done